### ISSUE TITLE
Can't install in Python2 due to missing README.md

### ISSUE THREAD

**Issue by polm** [SamuraiT/mecab-python3#21]
Trying to investigate #19, I got this error when installing in a python2 venv:

```
Collecting mecab-python3
  Using cached https://files.pythonhosted.org/packages/ac/48/295efe525df40cbc2173748eb869290e81a57e835bc41f6d3834fc5dad5f/mecab-python3-0.996.1.tar.gz
    Complete output from command python setup.py egg_info:
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/tmp/pip-install-4wqtTl/mecab-python3/setup.py", line 39, in <module>
        long_description = read_file('README.md'),
      File "/tmp/pip-install-4wqtTl/mecab-python3/setup.py", line 21, in read_file
        raw = open(filepath, 'rb').read()
    IOError: [Errno 2] No such file or directory: '/tmp/pip-install-4wqtTl/README.md'

    ----------------------------------------
Command "python setup.py egg_info" failed with error code 1 in /tmp/pip-install-4wqtTl/mecab-python3/

```

I see there were commits related to moving the README to rst and back again, maybe related to those and the timing of the Pypi release?

**Comment by ym8468** [comment]
I got the same issue when installing with pip install command:
```bash
[root@lab-node101 ~]# pip install mecab-python3
Collecting mecab-python3
  Downloading https://files.pythonhosted.org/packages/ac/48/295efe525df40cbc2173748eb869290e81a57e835bc41f6d3834fc5dad5f/mecab-python3-0.996.1.tar.gz
    Complete output from command python setup.py egg_info:
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/tmp/pip-build-K5bslS/mecab-python3/setup.py", line 39, in <module>
        long_description = read_file('README.md'),
      File "/tmp/pip-build-K5bslS/mecab-python3/setup.py", line 21, in read_file
        raw = open(filepath, 'rb').read()
    IOError: [Errno 2] No such file or directory: '/tmp/pip-build-K5bslS/README.md'

    ----------------------------------------
Command "python setup.py egg_info" failed with error code 1 in /tmp/pip-build-K5bslS/mecab-python3/
```

As my understanding "README.md" should be in the same directory as setup.py, so it should be:
/tmp/pip-build-K5bslS/mecab-python3/README.md
not
/tmp/pip-build-K5bslS/README.md

It seems there is bug in the read_file when getting the file path
```python
def read_file(filename):
    filepath = os.path.join(
        os.path.dirname(os.path.dirname(__file__)), filename)
```
This bug won't be triggered when running "python setup.py install" in current directory.

**Comment by zackw** [comment]
Huh, I thought it was _mandatory_ to run `setup.py` from the directory containing it.

A fix is in the works.

**Comment by zackw** [comment]
Please try the wheels and/or sdist at https://test.pypi.org/project/mecab-python3/0.996.2rc2/ , this problem should be corrected.  We plan to make a new release in the next couple of weeks.

**Comment by polm** [comment]
Verified the test pypi version works, thanks! :+1:

**Comment by zackw** [comment]
Fixed in 0.996.2.

**Comment by ym8468** [comment]
Installed version 0.996.2 by "pip install" with no problem. 
Thanks a lot for fixing this issue! @zackw
